### PR TITLE
Create initial draft of code of conduct

### PR DESCRIPTION
**Pull request by mellybelly**
ready for community editing and review.

### CODE REVIEW

File: docs/COC.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The OBO-COC team is intentionally small with only three members, in order to best support confidential handling of issues. Members of the OBO-COC team are nominated by the community and voted on by the OBO-Operations committee. The term shall be 2 years, but may be renewed.
+
+**Reporting issues** 
+
+If a community member experiences or witnesses unacceptable behavior, or if any other concern arises, then please report it by contacting a member of the OBO-COC team. If the issue involves a member of the COC-team, then please contact a different member. If these options are not sufficient, consider contacting a member of the OBO-Operations committee and/or your local ombuds office.
+All reports to OBO-COC will be handled with discretion and will receive serious attention. Reports should include the following:
+
+Name of person submitting the report.
+Names (real, nicknames, or pseudonyms) of any individuals involved. If there are additional witnesses, please include them as well. An account of the incident and whether it is ongoing. A link to any publicly available record (e.g., a mailing list archive, ticket, or a public Slack log), if applicable.
```
Comment (nlharris) [inline docs/COC.md]: This might be easier to read as a bullet list

**Comment by mellybelly** [comment]
Thanks Nomi. Yes it should sound familiar; its based on the many other CoCs I've written or contributed to for other events and organizations. I did try to tailor it for OBO-specific needs, but could do more on that front too. 

I believe there are some templates but still find you need to tailor for different contexts. I have had good success on a few occasions in the past when issues have arisen with some of this content, so its somewhat road-tested!

**Comment by matentzn** [comment]
Oh great initiative! I think the COC is sufficiently high level to not cause much disagreement - a bit like a Hippocratic Oath for Bio-ontologies :P

I think a conflict resolution team is a very cool idea! The more OBO community aligns in terms of standards and values, the more value we will have - exponentially for downstream tools and users.

**Comment by matentzn** [comment]
I saw some typos, like a double `if if` somewhere.

**Comment by yongqunh** [comment]
I remembered that Dr. Barry Smith introduced the code of ethics concepts for our ontologists, so I asked him about the detail.

Below is a slide that Barry presented
from International Conference on Biomedical Ontology<http://www2.unb.ca/csas/data/ws/icbo2013/> (ICBO), Montreal, Canada, July 8-9, 2013.
(“What Constitutes a Result in Ontology Research?” (Panel))
[image.png]
Thanks, Barry!

This may be useful for our discussion of the code of conduct.

Best,
Oliver

File: docs/COC.md (hunk 1 above)
Comment (cmungall) [reply docs/COC.md:50]: +1

**Comment by cmungall** [comment]
What are the remaining things to be done?

Do we need to first identify candidates to be on the team and then vote on them? Historically we have not been generally responsive, everyone is very busy. Are we ready to commit to this?

**Comment by matentzn** [comment]
What about raising this at OBO operations call to gather disagreement, the make another announcement on OBO discuss? I guess the question is whether every ontology in OBO automatically signs up for the COC (after it has been approved by the committee and left for two weeks for another round of feedback), or whether we get everyone to sign it explicitly.. I think the later would sort of be nice - even if it is a github signature :P

File: docs/COC.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+**Be considerate** 
+
+Your work will be used by other people, and you will depend on the work of others. Any decision you take will affect other community members. The consequences of your actions should be considered when making decisions. Remember that we're a large community, so you might not be communicating in someone else's primary language or share their area of expertise. This can present a challenge, but with thoughtful consideration, all communications between community members can be, and should be, considerate of the perspectives of others.
+
+**Be respectful** 
+
+Community members are likely to disagree on issues. This is to be expected (especially amongst ontologists!),  within our large, diverse group of individuals. However, we aim to support, at all times, constructive discourse when disagreements arise. This may lead to frustration from time to time, but we cannot allow frustration to turn into disrespectful behavior, mannerisms, or attacks. Our collective productivity very much depends on a supportive community, where people feel comfortable or not threatened when they need to disagree or propose alternative strategies.
```
Comment (pbuttigieg) [inline docs/COC.md:15]: This is key! We robustly disagree on the very nature of reality, but we're pretty cool about it and can still have beer (or a non-alcoholic beverage) with each other afterwards.

Comment (pbuttigieg) [inline docs/COC.md:15]: This is key! We robustly disagree on the very nature of reality, but we're pretty cool about it and can still have beer (or a non-alcoholic beverage) with each other afterwards.

**Comment by nlharris** [comment]
Other possibly relevant Codes of Conduct:
* Bioinformatics Community Conference (BCC2020, which will include BOSC and the Galaxy Community Conference): https://bcc2020.github.io/cod/
* The Carpentries: https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html

**Comment by pbuttigieg** [comment]
Some notes from today's Operations call:


### Regarding the CoC committee 

For initialisation, there should be a ToR for the CoC committee. Further:

* The founding CoC committee should have a first task of drafting some high-level SOPs on handling violations, mediations, or other concerns. 
* The CoC committee would harvest examples of SOPs from other communities we belong to (GSC, ESIP, etc)
* An escalation procedure and scope of the CoC's authority should be well-described (e.g. on the small chance things get that far, can they kick someone out of the consortium?)
  * The main focus of the CoC Committee should be conflict resolution / mediation, over penalisation 

Open questions:
* Does the CoC provide recommendations to the Operations Committee or does its decisions overrule it?
  * concerns of over powering a small group in the consortium
* Which categories of conflict are not addressable by this committee? Naturally, business shouldn’t extend into intellectual/ontological disagreement, but only if things get messy in a personal way. However, some fuzzy territory may exist, e.g. start of life for embryos - overlap of intellectual/scientific + religious/personal philosophy view

Alternative suggestions:
 * Rather than a standing CoC Committee, a mechanism to elect mediators as issues arise could be more maintainable / nimble. SOPs still needed

**Comment by lschriml** [comment]
National Microbiome Code of Conduct: https://microbiomedata.org/nmdc-code-of-conduct/
[see detailed on this link]
1. Be welcoming; 2. Be respectful; 3. Be supportive, positive, and professional.
--> I like their process for issue reporting:
 If you believe someone is violating the code of conduct, please report this in a timely manner by emailing support@microbiomedata.org or through an online form [here], with the option to report anonymously.

And the Center for Scientific Collaboration and Community Engagement: 
Diversity, Equity and Inclusion blog post series: 
https://www.cscce.org/resources/diversity-equity-and-inclusion-dei/
--> https://www.cscce.org/2018/06/04/community-guidelines-a-key-component-of-your-diversity-equity-inclusion-toolbox/

Cheers,
Lynn

**Comment by mellybelly** [comment]
Just putting a few comments here, though i think we should merge and start tickets with a new CoC tag? this thread is getting kinda wandering ..

1. while inventorying other CoCs is always a good idea, there has already been a lot of that in the creation of this draft. I think more important at this stage is whether or not this hits all the key points that we need for OBO.

2. Regarding a committee vs. obo-operations vs. emailing obo-discuss. Usually issues are things folks like to try to have confidential conversations about. I don't think it would be appropriate to email OBO-discuss unless there was already a public ticket or somesuch. I don't have strong feelings about whether obo-operations versus a committee, but in general providing a welcoming place to go with issues is important, and a large, open call may not be that. 

3. I think we could use the adoption of the CoC by all ontologies in the library as a key criteria. 

4. Once we have a CoC approved (it can certainly be versioned and enhanced over time), we can do some PR and get more folks involved if they see OBO as being a welcoming environment. 

Happy to join a future call to discuss further, but couldn't make the last one.
Thanks to everyone for helping with this!

**Comment by beckyjackson** [comment]
Discussed on 5/5/2020 - given that this has been open for a bit, if there are no further objections, we plan to merge this on 5/12/2020.

The next step is to determine how to vote for committee members - the first step would be to ask for volunteers. Then, we can submit a voting form to OBO-discuss based on these volunteers.

**Comment by matentzn** [comment]
I would like to discuss the nomination process for the COC in person during the next call, rather than on github.

File: docs/COC.md (hunk 1 above)
Comment (alanruttenberg) [reply docs/COC.md:50]: the comment about plagiarism doesn't seem relevent and might even be confusing given that we encourage reuse.

**Comment by ramonawalls** [comment]
This document was approved as the initial code of conduct for the OBO Foundry at the Operations Committee meeting on 5/19/2020. We expect that it will change over time, but it is a good start. We will begin the process of forming a CoC committee, and the committee will be tasked with developing procedures for handling complaints. The operations committee still needs to promote this CoC and encourage its adoption.

**Pull request merged**